lib/check-for-updates.js

```diff
@@ -26,7 +27,7 @@ module.exports = checkForUpdates;
 
 function readPackageJsonVersion () {
   return new Promise((resolve, reject) => {
-    Fs.readFile(__dirname + '/../package.json', (err, data) => {
+    Fs.readFile(Path.resolve(__dirname + '/../package.json'), (err, data) => {
```

Comment:
The nice thing about `path.resolve` is that we can pass multiple arguments instead of concatenating strings.

Ex: `Path.resolve(__dirname, '../package.json')`